package.json

```diff
@@ -12,9 +13,9 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "big-integer": "^1.6.26",
-    "dotenv": "^4.0.0",
-    "mongodb": "^2.2.33",
-    "twitter": "^1.7.1"
+    "big-integer": "1.6.26",
+    "dotenv": "4.0.0",
+    "mongodb": "2.2.33",
+    "twitter": "1.7.1"
```

Comment:
@natterstefan any reason why you want the exact versions?

Reply:
@philipheimboeck actually because I am used to it from work ^^.

The idea behind is to have a working version of the app with a given set of modules with a fixed version. So that newer versions of a package do not cause troubles potentially. Package updates must be done explicitly and do not happen "by accident".

Is okay for you?

Reply:
It is okay, but isn't that what a lock file is for? 😉